src/HomeWork_2/middlewares/check-authorisation.ts

```diff
@@ -0,0 +1,17 @@
+import { NextFunction, Request, Response } from 'express';
+import { jwtVerify } from 'jose';
+import { createSecretKey } from 'crypto';
+
+export const checkAuthorisation = async ({ headers }: Request, res: Response, next: NextFunction) => {
+  if (!headers.authorization) {
+    res.status(401).end();
+    return;
+  }
+  try {
+    await jwtVerify(headers.authorization, await createSecretKey(process.env.SECRET, 'utf-8'));
+  } catch (err) {
+    res.status(403).end();
```

Comment:
> In case of HTTPAuthorization headerhasinvalid JWTtoken in the request, the middleware should return HTTPcode 403(Forbidden Error) **and standard error message**.

Should be `res.sendStatus(403)`